tactics/redexpr.ml

```diff
@@ -126,7 +126,7 @@ type strategy_obj =
 let inStrategy : strategy_obj -> obj =
   declare_object {(default_object "STRATEGY") with
                     cache_function = (fun (_,obj) -> cache_strategy obj);
-                    load_function = (fun _ (_,obj) -> cache_strategy obj);
+                    open_function = (fun _ (_,obj) -> cache_strategy obj);
```

Comment:
I believe this is incorrect, you need to check if the integer is `1`. With your current patch, you activate the strategy as soon as one enclosing module is imported (even if not directly the one containing the command).

Reply:
Indeed changed